docs/add-ons/presentation.md

```diff
@@ -45,6 +45,28 @@ If you are in need of a guide on keeping a changelog, we would recommend checkin
 
 You can use own security profile for you Add-on with AppArmor. Default it is enabled and use the Docker default profile. Put `apparmor.txt` file into your Add-on folder and it will load this file as primary profile. Use the config options to set the name of that profile.
```

Comment:
```suggestion
You can use own security profile for you Add-on with AppArmor. By default it is enabled and uses the Docker default profile. Put a `apparmor.txt` file into your Add-on folder and it will load this file as primary profile. Use the config options to set the name of that profile.
```